### DISCUSSION TITLE
关于Button的Content属性的问题

### DISCUSSION THREAD

**Discussion by Zhaozhuofan** [BYJRK/DotNet-Discussions#248] [WPF 开发]
UP主您好,我在学习过程中对Button的Content属性产生了一些问题,希望您指点一下.
我在制作无边框的窗口的最大最小化及关闭使用了Button

<img width="125" alt="image" src="https://github.com/BYJRK/DotNet-Discussions/assets/103978682/84bf6d13-95fb-497e-94ba-4a89f7e871fd">

<img width="554" alt="image" src="https://github.com/BYJRK/DotNet-Discussions/assets/103978682/b42e15c2-0984-4c29-97d7-9129cc9a90da">

但是我希望最大化窗口之后最大化的图标产生一些变化,于是就这样写了代码

<img width="583" alt="image" src="https://github.com/BYJRK/DotNet-Discussions/assets/103978682/201730cc-2734-4d67-8121-1032f8d230ed">

但是实际效果却是最大化和还原后只显示了字符串,并没有显示出该字体对应的图标.
这该如何解决呢? 
<img width="168" alt="image" src="https://github.com/BYJRK/DotNet-Discussions/assets/103978682/2fc119b7-aa99-490d-b5ed-

最大化后:
<img width="155" alt="image" src="https://github.com/BYJRK/DotNet-Discussions/assets/103978682/2fe5ffc1-9474-4b09-bb85-7ad48e589275">

还原后:
<img width="153" alt="image" src="https://github.com/BYJRK/DotNet-Discussions/assets/103978682/31eedc44-a62f-4f6a-9b78-28820ee2704f">

这里是Button的样式
<img width="823" alt="image" src="https://github.com/BYJRK/DotNet-Discussions/assets/103978682/5521850d-306e-4313-98f1-3a9d23a52d88">

烦请大佬百忙之中指点一二

**Comment by KiyanYang** [comment]
在 XAML 中用 `&#x(UNICODE);`, 在 C# 中用 `\u(UNICODE)`, 例如

```xaml
<FontIcon Glyph="&#xE700;" />
```

```csharp
FontIcon icon = new FontIcon();
icon.Glyph = "\uE700";
```

https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.navigationviewitem.icon?view=winrt-22621

https://github.com/BYJRK/DotNet-Discussions/discussions/29

**Reply by Zhaozhuofan** [reply]
谢谢大佬,刚才试了一下,问题成功解决了
感谢大佬的指点^_^
